### PR TITLE
Cross-platform Wiimote App

### PR DESCRIPTION
**Pull request by cboulay**
I replaced the old Wiimote App with a new one.
Some important differences:

* Cross-platform Windows & Mac
* wiiuse is downloaded from github as needed so its library files don't have to be provided.
* Using a very minimal Qt GUI.

Other notes:

* This should be compatible with analysis scripts written for the old app. The XML descriptors are the same with the exception that the Wii Balance Board has 4 new channels (raw/uncalibrated data).
* The way wiiuse communicates with the controllers on Mac is a bit strange and the bluetooth calls would get out of sync with the controller state if run in a secondary thread. For this reason the controller is polled in the primary thread in a QTimer.

I used a shared computer for some of the commits and now they appear to have come from a student in the lab but questions should be directed to me.

### CODE REVIEW

**Comment by cboulay** [comment]
Oh, I should add that the build system does not work in Linux only because my Ubuntu is broken and I can't test. All code is using STL or other cross-platform libs so it's just a matter of modifying the cmake file to work in Linux.

**Comment by cboulay** [comment]
@mgrivich , I see you are going through old issues (thanks!) so I wanted to provide enough information so this can be fully compared to the existing app when deciding whether or not to merge.

I disabled the auto-enable of Wii Motion Plus and IR camera because one (or both) of these was causing problems for the wii balance board. These are enabled by default in the original version and therefore the balance board does not work properly in the original version. A fix for this could be as simple as setting them on/off depending on `wm->exp.type`. However, in the [wiiuse README](https://github.com/rpavlik/wiiuse#known-issues), it states that enabling Motion+ should only be done after polling for 10-15 seconds. The app currently doesn't have a mechanism to do that. Unfortunately I no longer have the Wii hardware so I can't test.

In summary (this table probably looks bad in e-mail, should be good in issue window):

|               | New Version    | Old Version    |
|---------------|----------------|----------------|
| Windows       | cmake          | MSVC 9         |
| Mac           | cmake          | No             |
| Linux         | Simple (?) fix | No             |
| Balance Board | OK             | Wrong          |
| IR Camera     | Simple fix     | OK             |
| Motion+       | Fix with delay | Fix with delay |
